src/shared/utils/kubeclient.ts

```diff
@@ -125,7 +125,12 @@ export const getKubeAccess = async (namespace: string, currentContext: string):
   if (hasErrors) {
     const errors = result.stderr;
     log.error(`get cluster access errors ${errors}`);
-    throw new Error(`Couldn't get cluster access for namespaces. Errors: ${JSON.stringify(errors)}`);
+    // Shall we create CustomErrors to handle different error types?
```

Comment:
Shall we handle situations like this with Custom Errors? @devcatalin @topliceanurazvan @mortada-codes

Reply:
this looks okay to me